modules/web/src/com/haulmont/cuba/web/gui/components/WebAbstractTable.java

```diff
@@ -2192,9 +2192,9 @@ public boolean saveSettings(Element element) {
         boolean settingsChanged = false;
 
         if (isUsePresentations()) {
-            boolean textSelection = component.isTextSelectionEnabled();
-            if (textSelection != Boolean.valueOf(element.attributeValue("textSelection"))) {
-                element.addAttribute("textSelection", String.valueOf(textSelection));
+            String textSelection = String.valueOf(component.isTextSelectionEnabled());
```

Comment:
Even though this solution resolves the issue, it would be preferable to compare **booleans**:
* First of all, because it's unclear without a comment why we need to compare strings
* To make sure that a setting string is a proper boolean value

I'd suggest just checking that `element.attributeValue("textSelection")` is `null`. If so, we need to add `"textSelection"` setting to the element, the same as if it's not `null` but doesn't equal to the table value.

Reply:
I've changed the fix according the recommendations